packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -161,6 +161,19 @@ describe('plugin-meetings', () => {
       assert.isFalse(audio.isSelf());
     });
 
+    describe('#isLocallyMuted()', () => {
+      it('does not consider remote mute status', async () => {
+        // simulate being already remote muted
+        meeting.remoteMuted = true;
```

Comment:
this only applies to audio, not video, but I think we don't support remote video mute right now, so I guess it's fine (maybe worth saying this in the comment?)

Reply:
Agreed @marcin-bazyl. Didn't realise it. Thanks for pointing. Have removed video related code for now. Lets revisit if time comes we have remote video mute by default.